docs/Technical Reference.md

```diff
@@ -2,110 +2,102 @@ This page provides a technical reference on certain advanced configuration optio
 
 ## Deploying the T4 Catalog on AWS
 
-The following section uses AWS CloudFormation to deploy and manage the resources required to run a T4 bucket and catalog in your own VPC.
+The following instructions use CloudFormation to install T4 on a bucket in your AWS account.
```

Comment:
Too limiting. We install more than half-a dozen services. `... use Cloudformation to deploy T4 services to your private AWS account.` It is a product detail, worth documenting for sure, that we currently work with _one_ bucket.